dist/pym.v1.js

```diff
@@ -128,7 +142,7 @@
      * @instance
      * @method autoInit
      */
-    lib.autoInit = function() {
+    lib.autoInit = function(ignoreEvent) {
```

Comment:
Name would be more clear if something like `doNotRaiseEvents`
`ignoreEvents` implies that it is ignoring events from another place